src/main/java/de/tum/in/www1/artemis/service/connectors/bamboo/BambooInternalUrlService.java

```diff
@@ -0,0 +1,20 @@
+package de.tum.in.www1.artemis.service.connectors.bamboo;
+
+import java.net.URL;
+import java.util.Optional;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Service;
+
+import de.tum.in.www1.artemis.service.InternalUrlService;
+
+@Profile("bamboo")
+@Service
+public class BambooInternalUrlService extends InternalUrlService {
```

Comment:
The `InternalUrlService` itself was already only for the "bamboo" profile 🤔 Why separate these?

Reply:
that was mistake, I removed the `@Profile` annotation from the `InternalUrlService`